ontology/cheminf.md

```diff
@@ -27,5 +27,3 @@ activity_status: active
 repository: https://github.com/semanticchemistry/semanticchemistry
 preferredPrefix: CHEMINF
 ---
-
-Includes terms for the descriptors commonly used in cheminformatics software applications and the algorithms which generate them.
```

Comment:
No objections (info is in struct data in line 16)

Reply:
yes, this is exactly the case for all ontologies in this PR. Thanks for the comment!